build.gradle

```diff
@@ -39,6 +39,7 @@ task downloadGsaLibFile(type: Download) {
 
 repositories {
     mavenCentral()
+    mavenLocal()
```

Comment:
It allows one to build htsjdk and pick up the updated artifact from your 
local maven repo.